examples/records_with_no_created_at_timestamp/connector.py

```diff
@@ -0,0 +1,76 @@
+# This is an example to show how to work with records where the source does not provide a created_at(or equivalent) field.
+# If you want to keep track of when the record was first observed, then the updated_at(or equivalent, eg, modified_at etc)
+# field can be made part of the composite Primary Key. By doing this you can ensure that successive syncs do not rewrite the
+# updated_at field in the existing row in the destination. Instead, it creates a new row with the new updated_at.
+# Now, the record with earliest value of _fivetran_synced system column represents when the record was first observed in source.
+# Refer to the Resulting table section at the bottom of this file for a detailed demonstration.
+# See the Technical Reference documentation (https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update)
+# and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details
+
+# Import required classes from fivetran_connector_sdk
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+
+def schema(configuration: dict):
+    return [
+        {
+            "table": "user",  # Name of the table in the destination.
+            "primary_key": ["id", "updated_at"],  # Primary key column(s) for the table.
+            "columns": {  # Define the columns and their data types.
+                "id": "INT",
+                "first_name": "STRING",  # String column for the first name.
+                "last_name": "STRING",  # String column for the last name.
+                "updated_at": "UTC_DATETIME",  # UTC date-time column for the updated_at.
+            },
+        }
+    ]
+# Define the update function, which is a required function, and is called by Fivetran during each sync.
+# See the technical reference documentation for more details on the update function
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update
+# The function takes two parameters:
+# - configuration: dictionary contains any secrets or payloads you configure when deploying the connector
+# - state: a dictionary contains whatever state you have chosen to checkpoint during the prior sync
+# The state dictionary is empty for the first sync or for any full re-sync
+def update(configuration: dict, state: dict):
+
+    # Represents a record fetched from source
+    row = {
```

Comment:
let's add 3 entries here
2 for normal record and 1 for an updated record

Reply:
@varundhall I added one more record in code. The third I have put it in the comments section at the bottom of the file. Can you please check if that seems appropriate?